### ISSUE TITLE
Putting Web Accessibility First - The History of the Web

### ISSUE THREAD

**Issue by studiomohawk** [frontendweekly/feeds#1881]
[Putting Web Accessibility First - The History of the Web](http://ift.tt/2sfHFCS) / Putting Web Accessibility First Quite frankly, my feeling is that the primary reason why the web is not accessible or not wholly accessible to people with&hellip;
